Added style for bordered panel row and lighter panel title weight

line47 opened the pull request:
Added style for bordered panel row and lighter panel title weight.

Commit:
Added style for bordered panel row and lighter panel title weight

msecret commented on src/css/components/panel.scss:
```diff
@@ -24,12 +24,20 @@ h4 + .panel {
-  font-weight: 600;
   margin-bottom: 2rem;
+
+  h1,
```
Currently, panel title is applied directly to the `h` tag, so this shouldn't be needed.

```
<h1 className={ this.styler('panel-title') }>{ this.props.title }</h1>
```

msecret commented on src/css/components/panel.scss:
```diff
@@ -80,6 +91,12 @@ h4 + .panel {
     padding: 1rem;
     width: 100%;
+
+    .route-form-preview {
```
This should actually go in the dashboard code:

https://github.com/18F/cg-dashboard/blob/067ef8fe56532ed2e032a18be9ad53b357d11586/static_src/css/route_form.css

I mean, technically it should go in cg-style, but we don't have a place for it yet, so it would do better there for now.

Commit:
Moved some code around

Currently, the route form stuff is in the CSS files in dashboard,
so moved that there.

The h rules weren't required because of the way we're using the
title class.
